### ISSUE TITLE
taxonomy upload issues

### ISSUE THREAD

**Issue by scratchpads** [NaturalHistoryMuseum/scratchpads2#5981]
- Added by [Dean Hendrickson](http://truchasmexicanas.myspecies.info/user/2)
- Scratchpad: [Truchas Mexicanas](http://truchasmexicanas.myspecies.info)

### Description:

I have a very small taxonomy in http://truchasmexicanas.myspecies.info. It's a bit unusual in that most of the taxa are undescribed, but IUCN agreed to list them in their Red List, and to do so they created a taxonomy based on vernacular names. Copying from their SIS database, here's a sample of the names they are using: 
Oncorhynchus chrysogaster_new
Oncorhynchus sp. nov. 'Acaponeta Trout'
Oncorhynchus sp. nov. 'Baluarte Trout'
Oncorhynchus sp. nov. 'Bavispe Trout'
Oncorhynchus sp. nov. 'Culiacán Golden Trout'
Oncorhynchus sp. nov. 'Mayo Trout'
Oncorhynchus sp. nov. 'Northern Conchos Trout'

I got those names into the Scratchpad site, but I consistently get failure reports when trying to import Excel files that have those names replacing binomials (as they do in IUCN's database). I also can't get Scratchpad to use lists of vernacular names. Importing Excel files with 2-5 vernacular names separated by | and language indicators like en|es|en|en|es, results in only 1 name or 2-5 copies of the first name without language being specified. Editing the taxonomy using the online tools also doesn't accept such vernacular names.

Thanks for your help.

**Comment by scratchpads** [comment]
Comment by [Dean Hendrickson](http://truchasmexicanas.myspecies.info/user/2)

I gave up on the Excel templates, and since it's such a small taxonomy, have focused on editing it using the internal tools provided. However, one taxon (O. clarkii) is problematic in that clicking on it for any reason takes the user to an error message, so I decided to simply delete it and re-enter it from scratch. However, my attempts to delete it in Structure / Taxa result in a very long time processing to eventually produce an AJAX error (500).
